### PR TITLE
Try to fix random all content grid

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #[issue number] by @[issue author]

## Description

This PR moves randomizing of the all content results to the Vuex media store. It uses only the first item for random seed, because the last item changes when load more button is clicked.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by zackkrida** [comment]
I can tell there's some major improvements in here to the structure of the data, but something is wrong with the logic and needs a little reworking, I'm only seeing one audio result, in the same position, per image page:

CleanShot 2022-01-20 at 21 01 33@2x

CleanShot 2022-01-20 at 21 01 35@2x

CleanShot 2022-01-20 at 21 01 37@2x

**Comment by zackkrida** [comment]
@obulat instead of working on this, it'd be better to look for bugs on the All Content grid on the main branch

File: src/components/VAudioTrack/VAudioTrack.vue [hunk 1]
```diff
@@ -91,7 +91,7 @@ export default defineComponent({
     size: {
       type: String,
       default: 'm',
-      validator: (val) => ['s', 'm', 'l'].includes(val),
+      validator: (val) => ['s', 'm', 'l', 'full'].includes(val),
```
Comment (dhruvkb) [inline src/components/VAudioTrack/VAudioTrack.vue:94]: What's the full size?

**Comment by obulat** [comment]
Closing this PR as this functionality has already been implemented and merged.

**Pull request closed without merging**